.circleci/config.yml

```diff
@@ -329,3 +429,18 @@ workflows:
       - deploy-production:
           requires:
             - hold-production
+
+  scheduled-smoke-test:
+    # triggers:
```

Comment:
will enable this once i can tell smoke test if succeeding and failing properly on master